ovs/lib/helpers/decorators.py

```diff
@@ -75,7 +75,33 @@ def new_function(*args, **kwargs):
     return wrap
 
 
-def ensure_single(task_name, extra_task_names=None, mode='DEFAULT', global_timeout=300):
+def ovs_task(**kwargs):
+    """
+    Decorator to execute celery tasks in OVS
+    These tasks can be wrapped additionally in the ensure single decorator
+    """
+    if 'name' not in kwargs:
+        raise ValueError('Name is a mandatory keyword argument for the ovs_task decorator')
+
```

Comment:
Wouldn't it be a good idea to add the `@wraps` decorator here, so we don't lose metadata